### ISSUE TITLE
Rails 4.1 routes

### ISSUE THREAD

**Issue by darrencauthon** [kilt-cms/kilt-cms#19]
I tried using Kilt in a Rails 4.1 RC1 application.  Everything worked except for one thing:  no routes were inserted into the application.

I think the problem is here:

https://github.com/AsheAvenue/Kilt/blob/1127de996a91bea8a6d5d0d453f21513b4fc71ec/lib/generators/kilt/backend_generator.rb#L15

``` ruby
inject_into_file Rails.root.join('config', 'routes.rb'), "\n\tmount Kilt::Engine => '/admin', as: 'kilt_engine'\n", :after => "#{Rails.application.class.parent_name.camelize}::Application.routes.draw do\n"      
```

The first line of my Rails 4.1 route file is:

``` ruby
Rails.application.routes.draw do
```

And my project name was "Blah".

Perhaps there should be a version check... or perhaps a call can be made for both the 4.0 and the 4.1 way.

**Comment by timboisvert** [comment]
We can just do it after "application.routes.draw".

**Comment by darrencauthon** [comment]
This is resolved in #27
